package.json

```diff
@@ -49,6 +49,10 @@
     "fast-xml-parser@^4.3.4": "^4.4.1",
     "ws@7.4.6": "^7.5.10"
   },
+  "dependencies": {
+    "react": "^18.2.0",
+    "react-native": "0.72.15"
+  },
```

Comment:
Hey @andrepimenta, I think we should keep the root `package.json` of our monorepo clear of dependencies since it’s not an actual application. However, we’re currently unable to get `storybook-react-native` working without defining them here. Do you know if there’s a way to avoid adding these dependencies to the root `package.json` and instead have them set as dependencies within `storybook-react-native`?